extensions/amp-a4a/0.1/refresh-intersection-observer-wrapper.js

```diff
@@ -65,16 +65,22 @@ export class RefreshIntersectionObserverWrapper {
     if (!this.viewportCallbacks_[refreshId]) {
       const viewportCallback = element.viewportCallback.bind(element);
       this.viewportCallbacks_[refreshId] = viewportCallback;
-      element.viewportCallback = inViewport => {
+      element.viewportCallback = (inViewport, propogateSignal = true) => {
         if (this.updateObserver_) {
           this.intersectionObserver_.tick(this.viewport_.getRect());
         }
-        viewportCallback(inViewport);
+        if (propogateSignal) {
+          viewportCallback(inViewport);
+        }
       };
     }
 
     this.updateObserver_ = true;
     this.intersectionObserver_.observe(element);
+    // Elements that appear and remain within the viewport for the duration of
+    // their existence may never have viewportCallback invoked. To ensure that
+    // refresh is triggered, we need to make this initial call.
```

Comment:
Can you elaborate on this comment a bit? Right now it's unclear to me what this is doing, as it looks like you're just adding  a call to viewportCallback that doesn't actually propagate the signal and I'm unclear where this new behavior is being consumed or how this works.

Reply:
Fixed so that now I'm calling `tick` directly, which should hopefully make this a bit clearer. Let me know otherwise.